packages/openapi-generator/src/cli.ts

```diff
@@ -60,12 +60,26 @@ export = class GenerateOpenApiClient extends Command {
       default: true,
       required: false
     }),
+    generateJs: flags.boolean({
+      name: 'generateJs',
+      description:
+        'By default, the generator will also generate transpiled .js, .js.map, .d.ts and .d.ts.map files. When set to false, the generator will only generate .ts files.',
+      default: true,
+      required: false
+    }),
     serviceMapping: flags.string({
       name: 'serviceMapping',
       description:
         'Configuration file to ensure consistent names between multiple generation runs with updated / changed metadata files. By default it will be read from the input directory as "service-mapping.json".',
       parse: input => resolve(input),
       required: false
+    }),
+    tsConfig: flags.string({
+      name: 'tsConfig',
+      description:
+        'tsconfig.json file to overwrite the default tsconfig.json. Existing keys will be overwritten new key will be added.".',
```

Comment:
what is the default tsconfig?

Reply:
It is some config I have copied from the OData case.